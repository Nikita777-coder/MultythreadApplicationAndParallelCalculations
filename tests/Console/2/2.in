10
9
4334243
75767
65765675
765757
8687679
656858568
5646465
8656568
9679679