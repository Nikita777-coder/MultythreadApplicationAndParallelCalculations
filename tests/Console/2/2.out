#######################Round#######################
	List of monahs before fight:
					Monah with ci = 9
					Monah with ci = 4334243
					Monah with ci = 75767
					Monah with ci = 65765675
					Monah with ci = 765757
					Monah with ci = 8687679
					Monah with ci = 656858568
					Monah with ci = 5646465
					Monah with ci = 8656568
					Monah with ci = 9679679

	Pairs are generated and:
			Monah with ci = 9 beat Monah with ci = 8687679
			Monah with ci = 8687679 was won
			Monah with ci = 75767 beat Monah with ci = 8656568
			Monah with ci = 8656568 was won
			Monah with ci = 765757 beat Monah with ci = 9679679
			Monah with ci = 9679679 was won
			Monah with ci = 656858568 beat Monah with ci = 9679679
			Monah with ci = 656858568 was won
			Monah with ci = 5646465 beat Monah with ci = 9679679
			Monah with ci = 9679679 was won

	List of monahs after fight:
					Monah with ci = 4334243
					Monah with ci = 65765675
					Monah with ci = 8687679
					Monah with ci = 656858568
					Monah with ci = 8656568



#######################Round#######################
	List of monahs before fight:
					Monah with ci = 4334243
					Monah with ci = 65765675
					Monah with ci = 8687679
					Monah with ci = 656858568
					Monah with ci = 8656568

	Pairs are generated and:
			Monah with ci = 4334243 beat Monah with ci = 8687679
			Monah with ci = 8687679 was won
			Monah with ci = 8687679 beat Monah with ci = 8656568
			Monah with ci = 8687679 was won
			Monah with ci = 8656568 wasn't fight that's why his power will increase in 2 times

	List of monahs after fight:
					Monah with ci = 65765675
					Monah with ci = 8687679
					Monah with ci = 656858568



#######################Round#######################
	List of monahs before fight:
					Monah with ci = 65765675
					Monah with ci = 8687679
					Monah with ci = 656858568

	Pairs are generated and:
			Monah with ci = 65765675 beat Monah with ci = 8687679
			Monah with ci = 65765675 was won
			Monah with ci = 656858568 wasn't fight that's why his power will increase in 2 times

	List of monahs after fight:
					Monah with ci = 65765675
					Monah with ci = 656858568



#######################Round#######################
	List of monahs before fight:
					Monah with ci = 65765675
					Monah with ci = 656858568

	Pairs are generated and:
			Monah with ci = 65765675 beat Monah with ci = 656858568
			Monah with ci = 656858568 was won

	List of monahs after fight:
					Monah with ci = 656858568



Monah with ci = 656858568 won!